examples/custom-config-ssl/configs/postgres-ha.yaml

```diff
@@ -0,0 +1,19 @@
+---
+bootstrap:
+  dcs:
+    postgresql:
+      parameters:
+        log_directory: pglogs
+        log_min_duration_statement: 0
+        log_statement: none
+        max_wal_senders: 6
+        shared_preload_libraries: pgaudit.so
+postgresql:  
+  pg_hba:
+    - local all postgres peer
+    - local all crunchyadm peer
+    - hostssl replication primaryuser 0.0.0.0/0 cert clientcert=1
+    - hostssl all primaryuser 0.0.0.0/0 reject
+    - hostssl all all 0.0.0.0/0 cert clientcert=1 map=ssl-testuser
+  pg_ident:
+    - ssl-testuser testuser@crunchydata.com testuser
```

Comment:
Is the example showing how to make a certificate with `cn=testuser@crunchydata.com`? If so, this works, otherwise, it's kind of silly to include the map.

If it's not showing how to do that, I would suggest we circle back up to make a map example that's more helpful, but we may do that anyway as we continue to advance our TLS enabled integration.

Reply:
The common name for the client certificate (`client.crt`) generated in the example is `testuser@crunchydata.com`, so the map is utilized when authenticating via that certificate:

https://github.com/CrunchyData/postgres-operator/blob/045820b21eab0a8b44b6515a4c856694fc8ecdeb/examples/custom-config-ssl/create.sh#L57

Reply:
Great. In the future we should update this example to be a bit more general purpose (the PostgreSQL docs have a good example). For now it's okay.